massdriver.yaml

```diff
@@ -2,7 +2,7 @@ schema: draft-07
 name: aws-eks-cluster
 description: Elastic Kubernetes Service is an open source container orchestration platform that automates many of the manual processes involved in deploying, managing, and scaling containerized applications.
 source_url: github.com/massdriver-cloud/aws-eks-cluster
-access: public
+access: private
```

Comment:
Assuming this was swapped to `private` for personal org testing, I think we want to swap back to `public` here.

Reply:
Asking bundles-dicsuss for design patterns here. 👍

Reply:
Alrighty, reverted 🤸